app/components/avo/fields/common/progress_bar_component.rb

```diff
@@ -1,17 +1,15 @@
 # frozen_string_literal: true
 
 class Avo::Fields::Common::ProgressBarComponent < Avo::BaseComponent
-  attr_reader :value
-  attr_reader :display_value
-  attr_reader :value_suffix
-  attr_reader :max
-  attr_reader :view
+  prop :value, Integer, reader: :public
+  prop :display_value, _Boolean, default: false, reader: :public
+  prop :value_suffix, _Nilable(String), reader: :public
+  prop :max, Integer, default: 100, reader: :public
+  prop :view, Symbol, default: :index, reader: :public do |value|
+    value&.to_sym
+  end
 
-  def initialize(value:, display_value: false, value_suffix: nil, max: 100, view: "index")
-    @value = value
-    @display_value = display_value
-    @value_suffix = value_suffix
-    @max = max
+  def after_initialize
     @view = Avo::ViewInquirer.new(view)
   end
```

Comment:
Can we merge the `view` initialization and make it as `Avo::ViewInquirer` type?

Reply:
done. Thanks for the suggestion